init-service/src/main/resources/application.yml

```diff
@@ -1,6 +1,9 @@
 spring:
   main:
     web-application-type: NONE # init-service is just a console app, should quit when done.
+azure:
+  activedirectory:
+    scope: api://mts-dev/default
```

Comment:
azure.activedirectory is something the azure libs are using, but this scope is an mts specific one.
Also, maybe worth to bring it from the config server and no to hard-code it here.

Reply:
agree, shouldnt this should be in config server?